### DISCUSSION TITLE
discordeno helper: the framework for discordeno

### DISCUSSION THREAD

**Discussion by Suya1671** [nestdotland/nest.land#284] [Show and tell]
Github: https://github.com/Suyashtnt/Discordeno-Helper
Nest.land entry: https://nest.land/package/discordeno-helper

Discordeno helper is a framework for discordeno to help making discord bots easier. nest.land is the offical place to get the stable library from because its that great.

**Comment by t8** [comment]
Very neat project, @Suyashtnt! What inspired you to create this?

CC @Skillz4Killz 😄

**Comment by Skillz4Killz** [comment]
Wow! Holy bonanza! Very cool!

Send a PR to add this to the docs of Discordeno so users can see your framework!!

**Comment by Suya1671** [comment]
When I started making bots I did it from scratch with discord.js. Not fun.
Then I learnt about discord-akairo and it made my life 10X easier. Then
deno came out and I instantly loved it, but there was nothing like
discord-akairo yet. So I decided to make something that is trying to do
everything it did but for Discordeno.
